app/cdap/components/OngoingDataExploration/LinearProgressBar/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { StyledLinearProgress } from './styles';
+
+const LinearProgressComponent = ({ progressValue }) => {
+  return <StyledLinearProgress variant="determinate" value={progressValue} />;
```

Comment:
Send variant and value from caller component
Make this component generic {...props}

give interface appropriately